off/off.go

```diff
@@ -0,0 +1,202 @@
+// Package off provides classes that write OFF files
+// OFF files store TES pulses projected into a linear basis
+// OFF files have a JSON header followed by a single newline
+// after the header records are written sequentially in little endian format
+// bytes		type			meaning
+// 0-3      int32     recordSamples (could be calculated from nearest neighbor pulses in princple)
+// 4-7      int32     recordPreSamples (could be calculated from nearest neighbor pulses in princple)
```

Comment:
I don't understand this or the previous misspelled parenthetical. How could the number of samples or presamples be computed in principle? I mean yes it can, but that presumes a certain set of rules for how the data are split into records. I think those rules should be treated as a plan that can be changed in Dastard from time to time, maybe even with user-settable parameters. In that event, we'll want the OFF file to record these values. (Yes, you could put into the header enough information to work out the rules, but ugh!)

Reply:
I'm assuming that you're using edgeMulti triggering with variable length records, and you could calculate the frames to previous pulse and frames to next pulse. But year, for now we'll include it. Really I should probably just put this text in the header.